lms/static/moment_requirejs.js

```diff
@@ -0,0 +1,15 @@
+(function(requirejs) {
```

Comment:
This is only used in lms-coffee suite so I am going to remove this in the LMS tests PR: https://github.com/edx/edx-platform/pull/12067